x/emissions/keeper/inference_synthesis/forecast_implied.go

```diff
@@ -38,6 +38,12 @@ func CalcForecastImpliedInferences(args CalcForecastImpliedInferencesArgs) (
 	err error,
 ) {
 	args.Logger.Debug("Calculating forecast-implied inferences", "topicId", args.TopicId)
+
+	// If NetworkCombinedLoss is nil, return empty maps immediately
+	if args.NetworkCombinedLoss == nil {
+		args.Logger.Debug("NetworkCombinedLoss is nil, returning empty forecast-implied inferences", "topicId", args.TopicId)
+		return make(map[Forecaster]*emissionstypes.Inference), args.InfererToRegret, args.ForecasterToRegret, nil
```

Comment:
```suggestion
		return nil, args.InfererToRegret, args.ForecasterToRegret, nil
```

May be an unnecessary allocation, a nil map has same behaviour than an empty one on read operations